addons/l10n_syscohada/wizard/account_move_reversal.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+from odoo import models
+
+
+class AccountMoveReversal(models.TransientModel):
+    """
+    Account move reversal wizard, it cancel an account move by reversing it.
+    """
+    _name = 'account.move.reversal'
+    _inherit = ['account.move.reversal']
```

Comment:
```suggestion
    _inherit = "account.move.reversal"
```
Same as above.
Also, please be consistent in the use of single/double quotes :slightly_smiling_face: